<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleSuppliesFormClass</class>
 <widget class="QDialog" name="BattleSuppliesFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BattleSuppliesForm</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Army 1 supplies per round: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Army1RoundSupplies">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Army 2 supplies per round: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Army2RoundSupplies">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>How many rounds add supplies for army 1: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Army1AmountRounds">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>How many rounds add supplies for army 1: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Army2AmountRounds">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Army 1 start supplies: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Army1supplies">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Army 2 start supplies: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Army2supplies">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ToMenu">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F0FFFF;
font-weight:800;
color:black;
</string>
   </property>
   <property name="text">
    <string>Start modeling</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplySupplies">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply supplies</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
